android/src/main/java/com/swmansion/gesturehandler/RNGestureHandlerPackage.kt

```diff
@@ -0,0 +1,29 @@
+package com.swmansion.gesturehandler
+
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.uimanager.ViewManager
+import com.facebook.soloader.SoLoader
+import com.swmansion.gesturehandler.BuildConfig
+import com.swmansion.gesturehandler.react.RNGestureHandlerModule
+import com.swmansion.gesturehandler.react.RNGestureHandlerRootViewManager
+import com.swmansion.gesturehandler.react.RNGestureHandlerButtonViewManager
+
+class RNGestureHandlerPackage : ReactPackage {
+  override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
+    if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
+      // For Fabric, we load c++ native library here, this triggers screen's Fabric
```

Comment:
"screens's"?

Reply:
Fixed in fe98498f3c9cdffd72a3b5803f82fe328bc33510